### DISCUSSION TITLE
Default numeric habits to target

### DISCUSSION THREAD

**Discussion by vsviridov** [iSoron/uhabits#998] [Feature Requests]
Currently when entering the numeric habit it always starts at 0.

It would be nice if there was an option to default it to target or to the last entered. It would speed things up.

Maybe additional buttons in the dialog?

![suggestion](https://user-images.githubusercontent.com/221684/124032679-b8ec5e00-d9ad-11eb-805f-cf0acf00b77b.png)

**Comment by ke352802081770314** [comment]
to be precise, it doesn't start with zero but with the value previously entered on that day. That's why it says "change value". Zero is the default for days with no input yet. 
What do you propose it should look like if a value was already entered on that day?
I like the quick-entry alternatives at the bottom.
But I am unsure when supporting this suggestion, that I might be considered supporting to default the entry field to a different value and I wouldn't want that.
